openmrs-19/src/main/java/org/motechproject/openmrs19/domain/OpenMRSPerson.java

```diff
@@ -161,8 +161,9 @@ public Boolean getBirthDateEstimated() {
         return birthDateEstimated;
     }
 
-    public Boolean isDead() {
-        return isDead;
+    // with getter isDead() task data source doesn't properly read this value
```

Comment:
Is it a problem in the Tasks module? If so, file a bug, append "TODO", and provide ticket ID.